Assets/Mirror/Editor/Weaver/Processors/SyncVarProcessor.cs

```diff
@@ -110,6 +110,23 @@ public static MethodDefinition ProcessSyncVarSet(TypeDefinition td, FieldDefinit
 
             ILProcessor setWorker = set.Body.GetILProcessor();
 
+
+            // if (syncVarEqual(value, ref playerData)) 
```

Comment:
```suggestion
            // if (syncVarEqual(value, ref playerData))
```
remove trailing space